dashboard/app/models/levels/craft.rb

```diff
@@ -405,6 +406,19 @@ def self.start_directions
     [['Up', 0], ['Right', 1], ['Down', 2], ['Left', 3]]
   end
 
+  def self.song_options
+    %w(
+      vignette1
+      vignette2-quiet
+      vignette3
+      vignette4-intro
+      vignette5-shortpiano
+      vignette7-funky-chirps-short
+      vignette8-free-play
+      nether2
```

Comment:
Note that we'll need the latest Craft update before Level Builders can start using this option.

Reply:
Yep, but note also that the `songs` attribute works as a whitelist of the values defined in `MUSIC_METADATA`, so we'll also need to update that when we update the Craft version, and until then using this option will be a nice quiet noop